client/rpc/finality/src/lib.rs

```diff
@@ -80,6 +88,16 @@ where
 			Ok(false)
 		}
 	}
+
+	async fn get_latest_block_hash(&self) -> RpcResult<H256> {
+		let res = self.backend.deref().latest_block_hash().await;
+		match res {
+			Ok(val) => Ok(val.into()),
+			Err(e) => {
+				Err(ErrorObject::owned(1, "No synced block", Some(format!("{:?}", e))).into())
+			}
```

Comment:
This could be improved:
```suggestion
			Ok(val) => Ok(val),
			Err(e) => Err(ErrorObject::owned(
			    jsonrpsee::types::error::UNKNOWN_ERROR_CODE, 
			    "No synced block",
			    Some(e),
			)),
```

Reply:
Chenged

Reply:
I think you did not push the changes

Reply:
sorry, something weird happend, tehre it is! :)